scss/detail.scss

```
@@ -34,133 +26,344 @@ $container-width: 900px;
-  position: absolute;
-  transform: rotate(15deg);
-  padding: 10px;
-  right: -5px;
-  top: 0px;
-}
-
-.header-detail {
-  align-items: center;
-  border-bottom: 4px solid $green-lemon;
-  display: flex;
-  padding-bottom: 12px;
-  width: 100%;
-}
-
-.title-book {
-  color: $black-default;
-  font-weight: 700;
-  margin-left: 8px;
-  font-size: 32px;
-  line-height: 32px;
-}
-
-.title-gender {
-  color: $grey;
-  font-weight: 700;
-  padding-left: 8px;
-  line-height: 28px;
-  font-size: 28px;
-}
 
-.detail-form {
-  margin: 36px 0 0 8px;
-}
-
-.detail-item {
-  color: $black-default;
-  font-weight: 700;
-  font-size: 20px;
-  line-height: 20px;
-}
-
-.detail-description {
-  color: $grey;
-  margin-left: 8px;
-  font-size: 20px;
-  line-height: 20px;
+  .container-img {
+    position: relative;
+    height: 100%;
+    max-width: 309px;
+    padding: 24px;
+    width: 100%;
+  
+    .badge {
+      display: flex;
+      height: $badge-height;
+      width: $badge-width;
+      position: absolute;
+      transform: rotate(15deg);
+      padding: 10px;
+      right: -5px;
+      top: 0px;
+    }
+  }
+  
+  .container-detail {
+    height: 100%;
+    padding: 24px;
+    width: 615px;
+  
+    .header-detail {
+      align-items: center;
+      border-bottom: 4px solid $green-lemon;
+      display: flex;
+      padding-bottom: 12px;
+      width: 100%;
+      
+      .title-book {
+        color: $black-default;
+        font-weight: 700;
+        margin-left: 8px;
+        font-size:  32px;
+        line-height: 32px;
+      }
+      
+      .title-gender {
+        color: $grey;
+        font-weight: 700;
+        padding-left: 8px;
+        font-size: 24px;
+        line-height: 24px;
+      }
+    }
+    
+    .detail-form {
+      margin: 36px 0 0 8px;
+    
+      .detail-item {
+        color: $black-default;
+        font-weight: 700;
+        font-size: 20px;
+        line-height: 20px;
+      }
+      
+      .detail-description {
+        color: $grey;
+        margin-left: 8px;
+        font-size: 20px;
+        line-height: 20px;
+      }
+    }
+  }
 }
 
 @media(max-width: $breakpoint) {
-  .back-arrow {
-    width: calc(#{$container-width} - 564px);
-    height: 20px;
-    margin-top: 33px;
-    justify-content: center;
+  .container{
     align-items: center;
+    background-color: $white-primary;
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    height: calc(#{$total-height} - 101px);
+    width: calc(#{$total-width} - 990px);
+    
+    .back-arrow {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      max-width: 1024px;
+      width: auto;
+      height: 20px;
+      margin-top: 33px;
+      cursor: pointer;
+    }
+    
+    .container-book {
+      flex-direction: column;
+      align-items: center;
+      height: 570px;
+      width: 336px;
+      
+      .container-img {
+        height: 337px;
+        padding: 38px 48px 0 48px;
```

You can use a shorthand here
```suggestion
        padding: 38px 48px 0;
```